inference_rules.adoc

```diff
@@ -821,6 +821,30 @@ The DLRMv2 MLPerf inference code has an option to aggregate multiple consecutive
 2. drawn uniformly from interval [`--samples-to-aggregate-min`, `--samples-to-aggregate-max`]	
 3. drawn from a custom distribution, with its quantile (inverse of CDP) specified in `--samples-to-aggregate-quantile-file=./tools/dist_quantile.txt`.
 
+=== LLM Benchmarks
+
+
+LLM benchmarks contain an auto-regressive generation loop. This loop’s execution is controlled by a decoding algorithm. The following rules discuss the decoding algorithm in the light of MLPerf Inference rules for the Closed submissions:
+
+* Decoding algorithm should follow the implementation available on huggingface: https://huggingface.co/blog/how-to-generate#beam-search 
+  ** We are using beam=4 for this algorithm’s setting.
```

Comment:
It's better to write that parameters of the decoding algorithm (in [https://github.com/mlcommons/inference/blob/master/language/gpt-j/backend.py#L13](https://github.com/mlcommons/inference/blob/master/language/gpt-j/backend.py#L13) ) must not be changed.